src/crimson/common/tri_mutex.h

```diff
@@ -51,7 +53,8 @@ class excl_lock_from_excl {
 /// independent of the conventional reader/writer lock usage. Here, what we
 /// mean is that we can pipeline reads, and we can pipeline writes, but we
 /// cannot allow a read while writes are in progress or a write while reads are
-/// in progress. Any rmw operation is therefore exclusive.
+/// in progress. Any rmw operation is therefore exclusive. Note that multiple
+/// writers are allowed by tri_mutex.
```

Comment:
please put this change in another commit. if we want to emphasis that multiple writes are allowed by `tri_mutex`, probably we could put this way:

> Unlike reader/write lock, tri_mutex does not enforce the exclusive access of write operations, on the contrary, multiple write operations are allowed to hold the same tri_mutex at the same time. Here, what we mean is that we can pipeline reads, and we can pipeline writes, but we cannot allow a read while writes are in progress or a write while reads are in progress.

Reply:
Splitted.

Reply:
could you reconsider the suggestion above?

Reply:
Yes, updated.